packages/EasyCodingStandard/src/Console/Style/EasyCodingStandardStyle.php

```diff
@@ -46,7 +46,7 @@ public function printErrors(array $errors): void
 
     /**
      * @param Error[] $errors
-     * @return string[]
+     * @return string[][]
```

Comment:
@muglug Why `string[][]`? :thinking:

Reply:
Because `$this->buildRow()` return an array of strings, so `$rows` will be an array of arrays of strings

Reply:
Yes, makes sense :clap:

@TomasVotruba Strange PHPStan didn't catch that

Reply:
Ticketed here: https://github.com/phpstan/phpstan/issues/747